aspnetcore/fundamentals/openapi/samples/9.x/BuildTime/csproj/MyTestApi.json

```diff
@@ -0,0 +1,57 @@
+{
+  "openapi": "3.0.1",
+  "info": {
+    "title": "GetDocument.Insider | v1",
```

Comment:
@captainsafia I can't reproduce this problem so take with a grain of salt. If someone else reports this let me know.
 As [this commit](https://github.com/dotnet/AspNetCore.Docs/pull/33361/commits/f8c51cb3212ecf2613235ac286c5bd2b82199e57#diff-fd505fc0f280268fcc635f15f56211857d7b8e444e1ba99ee4edbdbb1e52a74c) shows, after I ran `if (Assembly.GetEntryAssembly()?.GetName().Name != "GetDocument.Insider")` the first time, it changed the doc name from `MyTestApi` to `GetDocument.Insider` ***AND*** the change was sticky. I removed that code and and it continues to use the name `GetDocument.Insider`